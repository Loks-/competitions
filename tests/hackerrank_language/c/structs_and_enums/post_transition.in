2
A
2
2 1 3
a 2
b 3
1 2 4
c 2
B
1
4 1 4
d 1
e 2
f 3
h 4
5
3
2 B 0 A 1
3
1 A
1 B
