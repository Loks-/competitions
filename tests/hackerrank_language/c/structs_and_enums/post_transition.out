Town with the most number of packages is B
Town with the most number of packages is A
A:
	0:
		a
		b
	1:
		c
		e
		f
		h
B:
	0:
		d
